helpful.el

```diff
@@ -990,26 +1052,12 @@ state of the current symbol."
           (helpful--sym-value sym buf))
          "\n\n")
         (when (memq (helpful--sym-value helpful--sym buf) '(nil t))
-          (insert
-           (helpful--button
-            "Toggle"
-            'helpful-toggle-button
-            'symbol helpful--sym
-            'buffer buf)
-           " "))
-        (insert
-         (helpful--button
-          "Set"
-          'helpful-set-button
-          'symbol helpful--sym
-          'buffer buf))
+          (insert (helpful--make-toggle-button "Toggle" buf))
+          (insert " "))
+        (insert (helpful--make-set-button "Set" buf))
         (when (custom-variable-p helpful--sym)
-          (insert
-           " "
-           (helpful--button
-            "Customize"
-            'helpful-customize-button
-            'symbol helpful--sym)))))
+          (insert " ")
+          (insert (helpful--make-customize-button "Customize")))))
```

Comment:
You can pass multiple arguments to insert, rather than needing to call it repeatedly, so this can be `(insert " " (helpful--make-customize-button))`.

Reply:
I can take it out here, but I do think it's inelegant to require each new section to provide its own separator. A nicer way to do it (I'll put up another PR for it later if you think it's worth looking at) would be to do something like
```
(let* ((section-1 (when something
                    (make-section-1)))
       (section-2 (when something-else
                    (make-section-2)))
       (sections (-non-nil `(,section-1 ,section-2))))
  (insert (s-join " " sections)))
```
I think using multiple `insert`s is a tiny step towards separating the separator logic from the section content. But as I said, I can leave it out here.

(Btw, I used `suggest` to find both `-non-nil` and `s-join`!)

Reply:
We could certainly refactor later, but I'd leave it out here.

The approach of building sections is elegant, but the downside is that we don't show anything to the user until we've formatted all the sections. For code defined in C, or for variables with large values, some sections can be slower to render.